localflavor/no/forms.py

```diff
@@ -98,6 +98,66 @@ def _get_birthday(self, value):
         return birthday
 
 
+class NOBankAccountNumber(Field):
+    """
+    A form field for Norwegian bank account numbers.
+
+    Performs MOD11 with the custom weights for the Norwegian bank account numbers,
+    including a check for a remainder of 0, in which event the checksum is also 0.
+
+    Usually their string representation is along the lines of ZZZZ.YY.XXXXX, where the last X is the check digit.
+    They're always a total of 11 digits long, with 10 out of these 11 being the actual account number itself.
+
+    * Accepts, and strips, account numbers with extra spaces.
+    * Accepts, and strips, account numbers provided in form of XXXX.YY.XXXXX.
+
+    .. versionadded:: 1.4.2
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid Norwegian bank account number.'),
+        'invalid_checksum': _('Invalid control digit. Enter a valid Norwegian bank account number.'),
+        'invalid_length': _('Invalid length. Norwegian bank account numbers are 11 digits long.'),
+    }
+
+    def clean(self, value):
+        super(NOBankAccountNumber, self).clean(value)
+        if value in EMPTY_VALUES:
+            return ''
+
+        value = value.replace(".", "").replace(" ", "")
+        try:
+            # The control/check digit is the last digit
+            check_digit = int(value[-1])
+            bank_number = value[:-1]
+            if not bank_number.isdecimal():
+                raise ValueError
+        except ValueError:
+            raise ValidationError(self.error_messages['invalid'])
+
+        # They only have one length: the number is 10!
+        # That being said, you always store them with the check digit included.
+        if len(bank_number) is not 10:
+            raise ValidationError(self.error_messages['invalid_length'])
+
+        # Adhere for the special rule of these being "clearing" accounts
+        if bank_number[4:5] == "00":
```

Comment:
`bank_number[4:5]` can never be 2 characters. Did you mean to write `bank_number[4:6]`? I found this because I noticed that line 145 isn't covered by the tests.